src/BA_ABA_matrices.jl

```diff
@@ -16,7 +16,7 @@ Structure containing the BA matrix and other relevant data.
 - `ref_bus_positions::Set{Int}`:
         Set containing the indexes of the columns of the BA matrix corresponding
         to the refence buses
-- `radial_branches::RadialBranches`:
+- `radial_branches::RadialNetworkReduction`:
```

Comment:
To complete the renaming we should probably change all the field names too, right? Unless domain knowledge makes this non-confusing.
```suggestion
- `radial_network_reduction::RadialNetworkReduction`:
```